scala/scala_import.bzl

```diff
@@ -2,38 +2,16 @@ load("@io_bazel_rules_scala//scala:jars_to_labels.bzl", "JarsToLabelsInfo")
 load("//scala/settings:stamp_settings.bzl", "StampScalaImport")
 
 def _stamp_jar(ctx, jar):
-    stamped_jar_filename = "%s.stamp/%s-stamped.jar" % (ctx.label.name, jar.basename.rstrip(".jar"))
-
-    # Preferred way, but currently broken:
-    # java toolchain's ijar incorrectly handles MANIFEST sections
-    # https://github.com/bazelbuild/bazel/issues/12730
-    #    symlink_file = ctx.actions.declare_file(jar.basename)
-    #    ctx.actions.symlink(output = symlink_file, target_file = jar)
-    #    return java_common.stamp_jar(
-    #        actions = ctx.actions,
-    #        jar = symlink_file,
-    #        target_label = ctx.label,
-    #        java_toolchain = ctx.attr._java_toolchain[java_common.JavaToolchainInfo],
-    #    )
-
-    stamped_file = ctx.actions.declare_file(stamped_jar_filename)
-
-    ctx.actions.run(
-        executable = ctx.executable._ijar,
-        inputs = [jar],
-        outputs = [stamped_file],
-        arguments = [
-            "--nostrip_jar",
-            "--target_label",
-            str(ctx.label),
-            jar.path,
-            stamped_file.path,
-        ],
-        mnemonic = "StampWithIjar",
+    stamped_jar_filename = "%s.stamp/%s.jar" % (ctx.label.name, jar.basename.rstrip(".jar"))
```

Comment:
I guess there is no reason to `rstrip` jar suffix which is added back in `stamped_jar_filename`

Reply:
Thanks for catching this